src/Parser/BloodDeathKnight/Modules/Items/T20_4pc.js

```diff
@@ -25,17 +26,21 @@ class T20_4pc extends Module {
     const spellId = event.ability.guid;
     if (spellId === SPELLS.DEATH_STRIKE.id && !this.combatants.selected.hasBuff(SPELLS.GRAVEWARDEN.id, event.timestamp)) {
       this.deathStrinkeBuffless += 1;
+    }else{
+      if (spellId === SPELLS.DEATH_STRIKE.id) {
+        this.deathStrikeTotal += 1;
+      }
     }
-  }
 
+  }
```

Comment:
The whole event handler should be rewritten as 
```jsx
const spellId = event.ability.guid;
if (spellId !== SPELLS.DEATH_STRIKE.id) {
   return;
}
if (!this.combatants.selected.hasBuff(SPELLS.GRAVEWARDEN.id, event.timestamp)) {
   this.deathStrinkeBuffless += 1;
}
this.deathStrikeTotal += 1;
```

I think in your implementation, if you intend to show "X / Y casts ....", then Y should include also those X casts. That is - your `deathStrikeTotal` was in fact `buffedDeathStrikes` because it was in the else branch. Total Death Strikes should count those casts buffed and unbuffed.